pkg/microservice/aslan/core/code/service/project.go

```diff
@@ -99,6 +99,9 @@ func CodehostListProjects(codehostID int, namespace, namespaceType, keyword stri
 			if repo.Owner == nil || *repo.Owner.Login != namespace {
 				continue
 			}
+			if keyword != "" && !strings.Contains(repo.GetFullName(), keyword) {
```

Comment:
The github api is rate limited and is hard to visit inside the GW. So I think it is better to filter the result on frontend side.

Reply:
👌🏻